test/sanitizer_suppressions/lsan

```diff
@@ -0,0 +1,10 @@
+# addCoin(...) is used in the CoinSelection benchmark
+leak:addCoin
+
+# Suppress LSan warnings triggered in dependencies
+leak:libcrypto
+leak:libqminimal
+leak:libQt5Core
+leak:libQt5Gui
+leak:libQt5Widgets
+leak:new_allocator.h
```

Comment:
Should suppress the specific benchmark where this is happening, not the gcc internal

Reply:
This is the output I'm getting:

```
$ ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-6.0/bin/llvm-symbolizer ASAN_OPTIONS="" LSAN_OPTIONS="suppressions=$(pwd)/test/sanitizer_suppressions/lsan" src/bench/bench_bitcoin -evals=1 -scaling=0
# Benchmark, evals, iterations, total, min, max, median
…

=================================================================
==890==ERROR: LeakSanitizer: detected memory leaks

Indirect leak of 40040 byte(s) in 1001 object(s) allocated from:
    #0 0x535d60 in operator new(unsigned long) (/…/bitcoin/src/bench/bench_bitcoin+0x535d60)
    #1 0x5cd895 in __gnu_cxx::new_allocator<CTxOut>::allocate(unsigned long, void const*) /usr/bin/../lib/gcc/x86_64-linux-gnu/7.3.0/../../../../include/c++/7.3.0/ext/new_allocator.h:111:27
    #2 0x5cd895 in std::allocator_traits<std::allocator<CTxOut> >::allocate(std::allocator<CTxOut>&, unsigned long) /usr/bin/../lib/gcc/x86_64-linux-gnu/7.3.0/../../../../include/c++/7.3.0/bits/alloc_traits.h:436
    #3 0x5cd895 in std::_Vector_base<CTxOut, std::allocator<CTxOut> >::_M_allocate(unsigned long) /usr/bin/../lib/gcc/x86_64-linux-gnu/7.3.0/../../../../include/c++/7.3.0/bits/stl_vector.h:172
    #4 0x5cd895 in std::vector<CTxOut, std::allocator<CTxOut> >::_M_default_append(unsigned long) /usr/bin/../lib/gcc/x86_64-linux-gnu/7.3.0/../../../../include/c++/7.3.0/bits/vector.tcc:571

-----------------------------------------------------
Suppressions used:
  count      bytes template
   2002     392392 addCoin
-----------------------------------------------------

SUMMARY: AddressSanitizer: 40040 byte(s) leaked in 1001 allocation(s).
```

Note the absence of the benchmark in the symbolized stack trace.

I agree `leak:new_allocator.h` is not optimal but I don't know what to use instead :-)

Reply:
I've now adjusted the suppressions file. Now using `leak:__gnu_cxx::new_allocator<CTxOut>::allocate` which is a bit more specific. Please re-review :-)

Reply:
Would `leak:src/bench` work?

Reply:
@MarcoFalke I think so, but wouldn't that suppression be a bit too broad? Please advice :-)

Reply:
Fixed. Please re-review! :-)